default/scripting/ship_parts/Bombard/SP_CHAOS_WAVE.focs.txt

```diff
@@ -62,8 +89,36 @@ Part
                     ]
                     empire = Target.Owner
             ]
+
+        EffectsGroup    // monsters use chaos wave on player outposts
+            scope = NumberOf number = 1 condition = And [
+                Planet
+                InSystem id = Source.SystemID
+                OwnedBy affiliation = AnyEmpire
+                Stealth high = Source.Detection
+                Population high = 0
+            ]
+            activation = And [
+                Unowned
+                InSystem
+            ]
+            priority = [[POPULATION_DEFAULT_PRIORITY]]
+            effects = [
+                SetOwner empire = [[UNOWNED_EMPIRE_ID]]
+                RemoveSpecial name = "GAIA_SPECIAL"
```

Comment:
should maybe also mention that in the sitrep?

Reply:
Added separate message.